### PR TITLE
General fixes

### LINKED ISSUE #21: General Issues
- [x] Check all files 1 by 1, remove unused comments
- [x] Remove ``import React from 'react'`` where you have used it
- [x] Check paddings on all sections, you need to add more padding on some sections, check on figma
- [x] Add on mouse hover change text-color here on navigation items

Screenshot 2023-10-13 at 10 20 33

- [x] This should be with ``a href="/"`` like old website (the live one)

Screenshot 2023-10-13 at 10 21 06

- [x] These cards are too big, they should be square

Screenshot 2023-10-13 at 10 21 52

- [x] Copyright should be like this:

Screenshot 2023-10-13 at 10 24 55

- [x] Navigation is not like old live website, needs some improvements here

live:

Screenshot 2023-10-13 at 10 26 36

yours:
Screenshot 2023-10-13 at 10 27 23

- [x] Use this format for importing: 
``import Header from '@/components/Header'`` 
instead of ``import Header from '../Components/Header'``

### PR DESCRIPTION
**Pull request by Fetii1**
closes #21

### CODE REVIEW

File: components/Tabela.js [hunk 1]
```diff
@@ -1,38 +1,20 @@
-import React from 'react'
+import { programdata } from 'content/home'
```
Comment (Afan417) [inline components/Tabela.js:1]: rename to ``programs``, ene ke content agje

File: components/ImageComp.js [hunk 2]
```diff
@@ -1,39 +1,14 @@
-/* eslint-disable react/jsx-key */
-/* eslint-disable jsx-a11y/alt-text */
+import { homeImages } from 'content/home'
```
Comment (Afan417) [inline components/ImageComp.js:1]: rename to ``photos``

File: components/layout/FooterSection.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
 export default function FooterSection() {
   return (
-    <div className="w-full h-32 mt-24 bg-blue-950 flex items-end justify-center">
-      <p className="text-white text-xs mb-2">Copyright 2020. All rights reserved.</p>
+    <div className="w-full h-32 mt-32 bg-blue-950 flex items-end justify-center">
+      <p className="text-white text-xs mb-2 px-5 text-center">
+        © 2023 SHFK Forina. All Rights Reserved. Made with
+        <span className="text-red-400 text-lg"> ♥ </span> from
+        <Link target="_blank" href={'https://www.luckymedia.dev/'}>
+          <span className="font-bold"> Lucky Media</span>
```
Comment (Afan417) [inline components/layout/FooterSection.js:10]: empty space cektu, ``">   Lucky``

File: components/layout/Navigation.js [hunk 4]
```diff
@@ -21,11 +21,13 @@ export default function Navigation() {
...
       <div className="w-full h-6 bg-red-950"></div>
-      <div className="w-full shadow-md">
-        <div className="w-auto mx-3 sm:mx-10 flex flex-row justify-between flex-wrap">
-          <div className="flex w-full md:w-1/3 flex-row py-6 justify-between">
+      <div className="container shadow-y-md">
+        <div className="w-auto py-5 md:mx-3 sm:mx-10 flex flex-row justify-between flex-wrap">
+          <div className="flex w-full md:w-1/3 flex-row py-3 justify-between">
             <div className="w-full">
-              <p className="font-bold text-gray-800 text-2xl pl-5">SH.Q.K Forina</p>
+              <Link href={'/'}>
```
Comment (Afan417) [inline components/layout/Navigation.js:28]: dont need ``{ }``, use these brackets when you need something dynamic always

do href="/" here normal

File: components/sections/HeroSection.js [hunk 5]
```diff
@@ -12,7 +11,13 @@ export default function HeroSection() {
         height={1080}
         alt={'wave svg'}
       />
-      <Image src="/img/hero-image.png" width={1920} height={1080} alt={'Hero Image'} />
+      <Image
+        src="/img/hero-image.png"
+        width={1920}
+        height={1080}
+        alt={'Hero Image'}
```
Comment (Afan417) [inline components/sections/HeroSection.js:18]: normal,`` alt="hero image"``

File: package.json [hunk 6]
```diff
@@ -15,7 +15,8 @@
     "autoprefixer": "^10.4.13",
     "next": "^13.2.1",
     "next-seo": "^5.15.0",
-    "postcss": "8.4.21",
+    "node": "^20.7.0",
```
Comment (Afan417) [inline package.json:18]: pse e installut osht kjo ``node``?

Comment (Fetii1) [reply package.json:18]: ket se di. at dit si mundoheshim ta ndrecim chat card-in si nuk e mirte bg-orange ishte
tshof si ta fshej , nuk duhet kjo?

File: pages/404.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import Layout from '@/components/layout/Layout'
+import NotFound from '@/components/sections/NotFound'
+
+export default function Gallery() {
+  return (
+    <Layout title="Faqja Nuk u Gjet">
+      <NotFound />
+    </Layout>
```
Comment (Afan417) [inline pages/404.js:8]: ``NotFound`` wasnt needed to be a section but it's okay.

on error pages best practice is to put that code here, no need for ``NotFound``

but it's okay, keep it like this dont change it

File: components/Card.js [hunk 8]
```diff
@@ -1,13 +1,14 @@
-import React from 'react'
 import SVG from 'react-inlinesvg'
 
 function Card({ className, src, text }) {
   return (
-    <div className="md:w-1/3 my-2 w-full">
-      <div className={`${className} rounded-xl`}>
-        <div className="w-full h-auto py-14 flex flex-col items-center justify-center">
+    <div className="w-full xl:mb-16 md:-mb-20 lg:-mb-10 mb-0 my-2 md:max-w-xs">
```
Comment (Afan417) [inline components/Card.js:5]: so many margins, try to make this more simple

**Review by Afan417 — CHANGES_REQUESTED** [review]

**Pull request merged**